package.json

```diff
@@ -77,6 +77,7 @@
     ]
   },
   "dependencies": {
+    "@wesleytodd/openapi": "^0.1.0",
```

Comment:
I feel a bit hesitant to adopt "https://github.com/wesleytodd/express-openapi". There has been no updates for two years, e.g. no security patches are not applied and the adoption is low. 

On the positive side this looks like a small library. I suggest we fork it instead and take the extra cost of maintaining it our-self.